src/pages/components/Overlay.qml

```diff
@@ -549,9 +557,10 @@ Background {
                     }
                 }
 
-                onCloseAll: {
-                    webView.tabModel.clear()
+                onAboutToCloseAll: {
+                    pageStack.pop()
```

Comment:
This will change the behavior. Let me give an example of reading through a news website that has only 5 articles per week for free (quite normal pattern)

1) Open browser in normal mode
2) Go to tabs page
3) Switch to private browsing
4) Open your favorite news site to a tab
5) Open more tabs
6) Once done close all tabs

With this change in order to switch back to normal mode you need go back to the tabs page after you have closed all tabs to change back to normal browsing mode.

And yes, I do know that "Private browsing" is too hidden.

Reply:
I prefer keeping the old behavior of not moving between pages, keeping the remorse on tabs page, and keeping past tense.

Reply:
True, I'll revert to old behavior.